src/vim-loader/bfast.ts

```diff
@@ -55,4 +57,88 @@ class BFast {
   }
 }
 
-export { BFastHeader, BFast }
+export class BFastParser {
```

Comment:
There isn't any state on BFastParser other than logger, so we could just make them static functions on BFAST. Not important for now of course.

Reply:
True, it was mainly for symmetry with the other steps